app/serializers/v1/keyword_serializer.rb

```diff
@@ -3,5 +3,17 @@
 module V1
   class KeywordSerializer < ApplicationSerializer
     attributes :content, :status, :created_at, :updated_at
+
+    attributes :result_page_html, :links_count, if: proc { |_, params| params[:action] == :show }
+
+    attributes :search_entries_info, if: proc { |_, params| params[:action] == :show } do |_, params|
```

Comment:
Since serializers can be used in other endpoints (via associations), it is best to split the definition of serializer to avoid complexity and unexpected responses:
- 1 serializer for collections
- A serializer for single records 

You can check the following API endpoint implementation in https://github.com/nimblehq/the1-api/blob/development/engines/the1_reward/app/serializers/the1_reward/v2/reward_serializer.rb:
<img width="1136" alt="image" src="https://github.com/Goose97/google-scraper-ruby/assets/696529/483255f2-9efa-4760-9787-41bcfe088ca4">

Reply:
Fixed in c89ee2f160d3f8f4824711a6bea1b6972417dcfb